mivs/templates/emails/judging_reminder.txt

```diff
@@ -1,8 +1,8 @@
 {{ judge.attendee.first_name }},
 
-In our last email, we forgot to mention our timeline for when judging needs to be completed.  The final date we can accept judging is {{ c.JUDGING_DEADLINE|datetime_local }}.  However, we are encouraging judges to finish by {{ c.SOFT_JUDGING_DEADLINE|datetime_local }}, which will help us so that we can reassign any entries that still need judging.
+Don't forget! The final date we can accept judging is {{ c.JUDGING_DEADLINE|datetime_local }}.  The scores and feedback you submit are valued, so it's important to us and the developers that you're heard.
 
-{% if judge.judging_complete %}Our records indicate that you've completed reviews of all of games you've been assigned.  Thanks!  Please let us know if you're available to judge any additional games if any other judges find themselves unable to do so.{% else %}Our records indicate that you have completed judging on {{ judge.game_reviews|length }} games, out of the {{ judge.game_reviews|length }} you have been assigned.  Please try to get the remainder evaluated before the deadline, and if you don't think you'll be able to do so, please let us know so that we can assign those games to another judge.{% endif %}
+{% if judge.judging_complete %}Our records indicate that you've completed reviews of all of games you've been assigned.  Thanks!  Please let us know if you're available to judge any additional games, as some other judges may find themselves unable to do so.{% else %}Our records indicate that you have completed judging on {{ judge.game_reviews|length }} games, out of the {{ judge.game_reviews|length }} you have been assigned.  Please try to get the remainder evaluated before the deadline, and if you don't think you'll be able to do so, please let us know so that we can assign those games to another judge.{% endif %}
```

Comment:
I think the second `{{ judge.game_reviews|length }}` should be `{{ judge.reviews|length }}`. Looks like that was maybe an existing bug, but we should probably fix it while we're making changes here!